### PR TITLE
enhance(vorp_admin): add give gold + give money option to menu

### PR DESCRIPTION
**Pull request by kylemcshea**
This now has the menu showing:

1. Give Gold
2. Give Money

We no longer have a 0 { amount } or 1 { amount } to give out money to players via admin menu

### CODE REVIEW

File: server/resources/[VORP]/vorp_postman/Newtonsoft.Json.dll
Comment (kylemcshea) [inline server/resources/[VORP]/vorp_postman/Newtonsoft.Json.dll:1]: we need to delete this file

File: server/resources/[VORP]/vorp_stores/server/server.lua [hunk 1]
```diff
@@ -51,6 +51,7 @@ AddEventHandler('vorp_stores:sell', function(label, name, type, price, qty, stor
                     if items.amount >= quantity then
                         sellItems(_source,Character,ItemName,quantity,ItemLabel,total,total2,currencyType)
                         items.amount = items.amount-quantity --update amount left for store
+                        exports.webhook:send_to_splunk({_source = _source, char = Character, itemName = ItemName, quanity = quantity, ItemLabel = ItemLabel, total = total, total2 = total2, currencyType = currencyType},"sell")
```
Comment (kylemcshea) [inline server/resources/[VORP]/vorp_stores/server/server.lua:54]: we shouldnt push this splunk log in scope of the issue

File: .gitignore [hunk 2]
```diff
@@ -8,4 +8,6 @@ perms.cfg
 server.cfg
 vorp_perms.cfg
 txData
-.replxx_history
\ No newline at end of file
+.replxx_history
+start_6431_default.bat
```
Comment (kylemcshea) [inline .gitignore:12]: remove this replxx and other jazz

File: .gitignore [hunk 3]
```diff
@@ -7,5 +7,4 @@ CFX-files/*
 perms.cfg
 server.cfg
 vorp_perms.cfg
-txData
-.replxx_history
```
Comment (kylemcshea) [inline .gitignore:11]: yeah we wanted to remove replxx_history so this is right

File: server/resources/[VORP]/vorp_admin/client/database.lua [hunk 4]
```diff
@@ -549,3 +598,42 @@ function OpenInvnetory(inventorydata)
         end)
 
 end
+
+---@param targetData: { targetID: string, moneyType: 'money' | 'gold' }
+---@return void
+function giveCurrencyToPlayer(targetData)
```
Comment (kylemcshea) [inline server/resources/[VORP]/vorp_admin/client/database.lua:604]: we should remove this `giveCurrencyToPlayer` because its not being called anywhere

**Review by BuffsOverNexus — CHANGES_REQUESTED** [review]
A couple of things to address, but otherwise looks good!

**Comment by kylemcshea** [comment]
Merging because we resolved brian's requests

**Pull request merged**